internal/stacks/stackplan/planned_change.go

```diff
@@ -344,6 +349,64 @@ func (pc *PlannedChangeResourceInstancePlanned) PlannedChangeProto() (*terraform
 	}, nil
 }
 
+func encodePathSet(pathSet cty.PathSet) ([]*terraform1.AttributePath, error) {
+	if pathSet.Empty() {
+		return nil, nil
+	}
+
+	pathList := pathSet.List()
+	paths := make([]*terraform1.AttributePath, 0, len(pathList))
+
+	for _, path := range pathList {
+		jsonPath, err := encodePath(path)
+		if err != nil {
+			return nil, err
+		}
+		paths = append(paths, jsonPath)
+	}
+	return paths, nil
+}
+
+func encodePath(path cty.Path) (*terraform1.AttributePath, error) {
```

Comment:
It looks like there's some overlap with an existing conversion function in the `terraform1` package! https://github.com/hashicorp/terraform/blob/710f15fe8738e6a237f6f5ea40552d600ef420e0/internal/rpcapi/terraform1/conversion.go#L72-L74

Do we need both? From my quick skim they look pretty similar, but I defer to your judgment.

Reply:
Oh yeah good catch, I didn't see that, thanks for flagging 🙏